setup.cfg

```diff
@@ -79,8 +79,8 @@ install_requires=
     # Models and Metrics Extras
     transformers~=4.28.1  # For anthropic_client, huggingface_client, huggingface_tokenizer, test_openai_token_cost_estimator, model_summac (via summarization_metrics)
     # TODO: Upgrade torch
-    torch~=1.12.1  # For huggingface_client, yalm_tokenizer, model_summac (via summarization_metrics)
-    torchvision~=0.13.1  # For huggingface_client, yalm_tokenizer, model_summac (via summarization_metrics)
+    torch>=1.12.1  # For huggingface_client, yalm_tokenizer, model_summac (via summarization_metrics)
```

Comment:
how about `torch>=1.12.1,<2.1.0` so things don't break when torch 3 or 2.1 come out? same for torchvision.

Reply:
I think minor version upgrades should be fine assuming PyTorch follows semantic versioning. We were doing `torch~=1.12.1` before which allows minor upgrades. It should probably be `torch~=1.12.1,torch~=2.0.0` though.

Reply:
yeah, that's good to me---as long as things don't blow up when torch 3 gets released.

Reply:
Thanks @nelson-liu !